### ISSUE TITLE
Vulnerability report

### ISSUE THREAD

**Issue by JafarAkhondali** [scaljeri/chess.com#63]
We are a group of researchers from Leiden University, and we conduct research on vulnerabilities in open-source software. We have discovered and verified a high-severity vulnerability in your project(scaljeri/chess.com). Explaining the vulnerability further in this issue could allow malicious users to access details, so we recommend [enabling private vulnerability reporting on GitHub](https://docs.github.com/en/code-security/security-advisories/working-with-repository-security-advisories/configuring-private-vulnerability-reporting-for-a-repository) to discuss this matter confidentially.
        After you have enabled this feature, please add a comment to this issue so we can continue our discussion. If you have any questions, feel free to leave a reply here or send an email to: j.akhoundali [at] liacs.leidenuniv.nl

**Comment by have-no-clue-what-im-doing** [comment]
Feel like y'all got bigger fish to fry than focusing on a little chess bot 🤪

**Comment by scaljeri** [comment]
you are correct, this project belongs to the past :)

**Comment by JafarAkhondali** [comment]
@scaljeri Yes this is an old project, but let us know if you would to receive the report.
